pipestat/parsed_schema.py

```diff
@@ -423,3 +261,37 @@ def _recursively_replace_custom_types(s: Dict[str, Any]) -> Dict[str, Any]:
         spec.setdefault("required", []).extend(curr_type_spec["required"])
         spec[SCHEMA_TYPE_KEY] = curr_type_spec[SCHEMA_TYPE_KEY]
     return s
+
+
+def replace_JSON_refs(s: Dict[str, Any], data: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
Do we really have to replace refs in the schema? Shouldn't the JSON-schema implementation do this for us? This surprises me